File: Assets/Scripts/_roundCounter.js
var NewRound : AudioClip;
var BossRound : AudioClip;


function PlayNewRound ()
{

    GetComponent.<AudioSource>().PlayOneShot(NewRound);
}
function PlayBossRound ()
{

    GetComponent.<AudioSource>().PlayOneShot(BossRound);
}

File: Assets/Scripts/_roundCounter.cs
using UnityEngine;
using System.Collections;

public class _roundCounter : MonoBehaviour
{
    AudioClip NewRound;
    AudioClip BossRound;


    void PlayNewRound (){

        GetComponent<AudioSource>().PlayOneShot(NewRound);
    }
    void PlayBossRound (){

        GetComponent<AudioSource>().PlayOneShot(BossRound);
    }
}